org.cryptomator.Cryptomator.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
Should be upstreamed

Reply:
Agreed...and I think Stenzel would agree also. If I can sort out that FUSE problem and the Cryptomator folks move forward with the Flatpak, we can place these three files in the Linux repo.